src/components/Button/Button.tsx

```diff
@@ -0,0 +1,16 @@
+import { FC } from 'react';
+
+type Props = React.ButtonHTMLAttributes<HTMLButtonElement> & {
+  children: React.ReactNode;
+};
+
+export const Button: FC<Props> = ({ children, ...props }) => {
```

Comment:
Why you need Button wrapper?

Reply:
simple practice to move some common logic

Reply:
good job with separate component for buttons!